src/components/marker-chart/Canvas.js

```diff
@@ -351,7 +349,7 @@ class MarkerChartCanvas extends PureComponent {
     ctx.fillRect(x + c, bottom - c, width - 2 * c, c);
   }
 
-  getHoveredMarkerInfo(hoveredItem: IndexIntoMarkerTiming): React$Element<*> {
+  getHoveredMarkerInfo(hoveredItem: IndexIntoMarkerTiming): React.Element<any> {
```

Comment:
See [the documentation for React.Element](https://flow.org/en/docs/react/types/#toc-react-element)

Reply:
Thanks for the detailed comments on this.

Reply:
I added a comment in the code too.